x-pack/plugins/watcher/server/routes/api/register_load_history_route.ts

```diff
@@ -29,7 +29,7 @@ function fetchHistoryItem(dataClient: IScopedClusterClient, watchHistoryItemId:
         },
       },
     })
-    .then(({ body }) => body);
```

Comment:
I'm wondering if we could delete the line `.then((body) => body);` in more places similar to the file `x-pack/plugins/watcher/server/routes/api/settings/register_load_route.ts`

Reply:
Thanks a lot!